docs/contributing/devguide.md

```diff
@@ -47,7 +47,7 @@ Building the operator binary and docker image:
 
 ```shell
 make build
-make docker.build IMG=external-secrets:latest
+DOCKER_BUILDKIT=1 make docker.build IMAGE_NAME=external-secrets IMAGE_TAG=latest
```

Comment:
It would be better if we set `DOCKER_BUILDKIT=1` in the Makefile since engineers don't have to care about it. Also, you are right and the `IMG` actually does nothing, and we can just remove it 🙂 

Those changes are not directly related to the PR itself, so you can just revert the changes on this file if you don't want to work on the comment above!